test/e2e/testutils.go

```diff
@@ -0,0 +1,132 @@
+package e2e
```

Comment:
Why don't we use the `helpers.go` file for this? Why do we need a second utilities/helpers file?

Reply:
Good point - I'll move any methods into the existing `helpers.go` file

Reply:
Moved.